### PR TITLE
chore(j-s): Use GraphQL generated Gender type

### PR DESCRIPTION
**Pull request by oddsson**
# Use GraphQL generated Gender type in the front end

[Asana](https://app.asana.com/0/1199153462262248/1204890034208750/f)

## What

Use GraphQL generated Gender type in the front end

## Why

We are slowly moving towards using only types generated by GraphQL in the front end.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Court/Indictments/Defender/SelectDefender.tsx [hunk 1]
```diff
@@ -10,7 +10,8 @@ import {
 import { FormContext } from '@island.is/judicial-system-web/src/components/FormProvider/FormProvider'
 import { core } from '@island.is/judicial-system-web/messages'
 import { capitalize } from '@island.is/judicial-system/formatters'
-import { Defendant, UpdateDefendant } from '@island.is/judicial-system/types'
+import { TempUpdateDefendant as UpdateDefendant } from '@island.is/judicial-system-web/src/types'
```
Comment (unakb) [inline apps/judicial-system/web/src/routes/Court/Indictments/Defender/SelectDefender.tsx:13]: Will you eventually be using the generated UpdateDefendantInput here?

**Review by unakb — APPROVED** [review]
Nice, love where this is heading! 🧹

File: apps/judicial-system/web/src/routes/Court/Indictments/Defender/SelectDefender.tsx (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Court/Indictments/Defender/SelectDefender.tsx]: No, probably not! I changed this to use generated `UpdateDefendantInput` and `CreateDefendantInput` instead. Could you have another look?

**Review by unakb — APPROVED** [review]
Really nice cleanup, tested locally as well and it worked as expected 😊

**Pull request merged**